src/Couchbase.Lite.Tests.NetCore/Couchbase.Lite.Tests.NetCore.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
       <DefineConstants>DEBUG;LITECORE_PACKAGED;COUCHBASE_ENTERPRISE</DefineConstants>
-      <WarningLevel>0</WarningLevel>
+      <WarningLevel>4</WarningLevel>

```

Comment:
Don't know what is this for, but is this change by intention?

Reply:
I don't know how it got changed to 0, but it was disabling all warnings.  Without this it would not show deprecation warnings.